generated/nifake/custom_struct.py

```diff
@@ -0,0 +1,33 @@
+import ctypes
```

Comment:
This should not be here... it's from a different PR!

Reply:
Removed